migracja.rst

```diff
@@ -0,0 +1,235 @@
+**Laboratorium EE**
+
+*29 XII 2016*
+
+Model migracji z systemów EZD PUW i eDok
+========================================
+
+Wstęp
+<<<<<
+
+Operacja migracji danych z instalacji istniejącego systemu EZD do innego system EZD, powinna być
+realizowana za pośrednictwem pliku pośredniego (lub pewnej liczby plików) — a nie on-line.
+
+Dane pośrednie będą zapisane w formacie, który jest wspólny dla obu systemów.
+
+Migracja a eksport wybranych danych
+<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+Przekazywanie danych z systemu EZD może mieć miejsce w kilku sytuacjach. Każdy system EZD musi mieć
+zaimplementowane operacje eksportu (przekazywania) danych do innych systemów. Przynajmniej:
+
+- przesyłanie dokumentów do innych systemów teleinformatycznych, w szczególności przez eksport
+  dokumentów i ich metadanych z zachowaniem powiązań pomiędzy tymi dokumentami i metadanymi
+  zob. §6 pkt 14 Rozporządzenia Ministra Spraw Wewnętrznych i Administracji z dnia 30 października
+  2006 r. w sprawie szczegółowego sposobu postępowania z dokumentami elektronicznymi
+  http://isap.sejm.gov.pl/DetailsServlet?id=WDU20062061518)
+- wytworzenie „paczki archiwalnej” z dokumentami, do przekazania do Archiwum Państwowego oraz spisu
+  zdawczo-odbiorczego odpowiadającego określonej paczce archiwalnej (zob. §6 pkt 13 14
+  Rozporządzenia MSWiA z dnia 30 października 2006)
+
+Do marca 2017r., jest też realizowany projekt we współpracy MC - COI - NSA - WSA - PUW, mający na celu
+przygotowanie formatu danych, który będzie używany do przekazywania danych z systemów EZD
+do systemu do obsługi postępowania sądowoadministracyjnego realizowanego w ramach projektu OPSAD [1]_.
+
+Wydaje się rozsądnym, by do wszystkich operacji masowego produkowania danych do wymiany, używany
+był syntaktycznie ten sam format danych — a tworzone pliki różniły się tylko zakresem danych,
+w zależności od zamierzonego celu eksportu.
+
+Model danych
+<<<<<<<<<<<<
+
+Dane przechowywane w systemie
+-----------------------------
+  
+Rysunek poniżej ilustruje fakt, iż przeniesienie danych z jednego systemu do drugiego jest możliwe
+tylko dla danych takich, które są modelowane w obydwu systemach i są dla systemu docelowego
+"interesujące" (użytkownicy systemu potrzebują takich danych w swej działalności).
+
+.. image:: images/dane_dwoch_systemow.svg
+
+To, że w jednym systemie nie ma pewnych danych, które są obecne w innym systemie może wynikać
+z braku odpowiednich pól albo z ograniczeń zastosowanego modelu danych.
+
+Można też rozróżnić dane które mogą być w jakimś systemie gromadzone i przechowywane, od danych
+których potrzebują użytkownicy do realizacji swoich zadań. (Znane są przykłady systemów, gdzie
```

Comment:
Pozwolę się nie zgodzić. Myślę, że to jest przypadek ujęty w punkcie
"97.1.
W nawiasy ujmujemy te części wypowiedzi, które uzupełniają lub objaśniają tekst główny..."
Bez nawiasów, to zdanie wyglądałoby na zbyt ważne.